static/docker-compose.yml

```diff
@@ -64,6 +64,13 @@ services:
       PG_RUNBOOKS_RW_PASSWORD: ${PG_RUNBOOKS_RW_PASSWORD:?err}
       PG_RUNBOOKS_RW_USER: ${PG_RUNBOOKS_RW_USER:?err}
       PG_RUNBOOKS_DB: ${PG_RUNBOOKS_DB:?err}
+      PG_CKEDITOR_ADMIN_PASSWORD: ${PG_CKEDITOR_ADMIN_PASSWORD:?err}
+      PG_CKEDITOR_ADMIN_USER: ${PG_CKEDITOR_ADMIN_USER:?err}
+      PG_CKEDITOR_DB: ${PG_CKEDITOR_DB:?err}
+      PG_CKEDITOR_RO_PASSWORD: ${PG_CKEDITOR_RO_PASSWORD:?err}
+      PG_CKEDITOR_RO_USER: ${PG_CKEDITOR_RO_USER:?err}
+      PG_CKEDITOR_RW_PASSWORD: ${PG_CKEDITOR_RW_PASSWORD:?err}
+      PG_CKEDITOR_RW_USER: ${PG_CKEDITOR_RW_USER:?err}
```

Comment:
Yes, because the plan is to run a migration that reads these variables and configures the database, that way, the configuration between a new instance that creates the DB with the init script is the same as an existing instance that creates the DB with a migration